llvm/test/CodeGen/DirectX/ShaderFlags/max-64-uavs-array-sm6_5.ll

```diff
@@ -29,5 +29,10 @@ define void @test() "hlsl.export" {
   ret void
 }
 
+; Set dx.valver and dx.resmayalias to prevent flags ResMayNotAlias and
+; UAVsAtEveryStage from being set, as to not distract from the flag that is
+; actually being tested
```

Comment:
I don't think we need this comment in every test file. Just mention this in the PR description / commit message and people can find it in the git history if they're trying to figure out why valver was added here.